aiida/orm/implementation/sqlalchemy/groups.py

```diff
@@ -124,6 +124,15 @@ def store(self):
         self._dbmodel.save()
         return self
 
+    def count(self):
+        """Return the number of entities in this group.
+
+        :return: integer number of entities contained within the group
+        """
+        from aiida.backends.sqlalchemy import get_scoped_session
+        session = get_scoped_session()
+        return session.query(self.MODEL_CLASS).join(self.MODEL_CLASS.dbnodes).filter(DbGroup.id == self.pk).count()
```

Comment:
Can't this be just piped to len(self.nodes)? This, below, should do the same thing? (see __len__ in the property returned by `nodes`?

Reply:
That's what I was originally using, but at least for the SqlA implementation it was a factor 8 slower on my production database

Reply:
ok